extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -44,6 +49,50 @@ export class EntitlementStore {
 
   }
 
+  /**
+   * Resolves a platform in the store
+   * @param {string} serviceId
+   * @param {!./subscription-platform.SubscriptionPlatform} platform
+   */
+  resolvePlatform(serviceId, platform) {
+    this.subscriptionPlatforms_[serviceId] = platform;
+  }
+
+  /**
+   * Returns the platform for the given id
+   * @param {string} servideId
+   * @returns {!./subscription-platform.SubscriptionPlatform}
+   */
+  getPlatform(servideId) {
+    const platform = this.subscriptionPlatforms_[servideId];
+    dev().assert(platform, `Platform for id ${servideId} is not resolved`);
+    return platform;
+  }
+
+  /**
+   * Returns the local platform;
+   * @returns {!./subscription-platform.SubscriptionPlatform}
+   */
+  getLocalPlatform() {
+    return this.getPlatform('local');
+  }
+
+  /**
+   * Returns all the platforms;
+   * @returns {!Array<!./subscription-platform.SubscriptionPlatform>}
+   */
+  getAllResolvedPlatforms() {
+    const platforms = [];
+    for (const platformKey in this.subscriptionPlatforms_) {
+      if (this.subscriptionPlatforms_.hasOwnProperty(platformKey)) {
```

Comment:
No need for hasOwnProperty here. But if you'd like, you should use `hasOwn` from `object.js` utils to avoid compatibility issues.